src/components/DpTabs/DpTabs.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import type { Meta, StoryObj } from '@storybook/vue'
 import DpTabs from './'
-import DpTab from './'
+import DpTab from './DpTab.vue'
```

Comment:
why do we need a explicit import for DpTab but not for DpTabs

Reply:
DpTabs imported from index.ts
```
export { default } from './DpTabs.vue'
export * from './DpTabs.vue'
```

Reply:
ok. So we could do it for tab.vue the same way?

Reply:
there can only ever be one default export!

Reply:
@spiess-demos

Reply:
as we need both components within the same story, but the `import x from ./` syntax only works for default exports, which we cannot have two at the same time within one file, the solution `import DpTab from './DpTab.vue'` is ok imho.